skimage/registration/masked_register_translation.py

```diff
@@ -15,31 +15,26 @@
 from .._shared.fft import fftmodule, next_fast_len
 
 
-def masked_register_translation(
-        src_image,
-        target_image,
-        src_mask,
-        target_mask=None,
-        overlap_ratio=3 / 10):
-    """
-    Masked image translation registration by masked normalized
+def masked_register_translation(reference_image, moving_image, reference_mask,
+                                moving_mask=None, overlap_ratio=0.3):
+    """Masked image translation registration by masked normalized
     cross-correlation.

```

Comment:
```suggestion
def masked_register_translation(reference_image, moving_image, *, reference_mask,
                                moving_mask=None, overlap_ratio=0.3):
    """Image translation registration by masked normalized cross-correlation.
```

Reply:
Should `reference_mask` be a required named argument?

Reply:
@jni, `reference_mask` is now a required named argument.